### PR TITLE
Update getting-started

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

This is ~~big~~ ~~huge~~ enourmous! It also changes a lot of stuff... I'll try to summarise here:

- Changes the structure of all getting started pages
   - These are now based on what you install on and then sectioned into each supported installation type for that platform.
- Adds a new `tabbed_block` plugin.
- Adds a new Common tasks section for all installation types.
- /hassio is removed and the content has moved to more logical places with this new structure.
- I _think_ I've added redirects for all removed/moved pages.

Preview: https://deploy-preview-16039--home-assistant-docs.netlify.app/

_Sorry! :cry:_

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/getting-started/index.markdown [hunk 1]
```diff
@@ -1,86 +1,14 @@
 ---
-title: "Install Home Assistant"
-description: "Getting started: How to install Home Assistant."
+title: "Getting Started"
+description: "Getting started with Home Assistant"
+hide_github_edit: true
```
Comment (frenck) [inline source/getting-started/index.markdown]:
```suggestion
feedback: false
```

**Review by frenck — DISMISSED** [review]

File: source/getting-started/index.markdown (hunk 1 above)
Comment (ludeeus) [reply source/getting-started/index.markdown]: This actually does not make sense to hide here, we want feeedback :smile: 
```suggestion
```

Comment (frenck) [reply source/getting-started/index.markdown]: Even better :)

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
I think this is good enough to merge and work from that. The structure is the important thing in this PR.

There are some things that need improvement, but those are the result of the original guides used (and thus not introduced in this PR).

Let's get something moving 👍 

Nice work @ludeeus!

**Pull request merged**